aws/structure.go

```diff
@@ -5522,3 +5523,18 @@ func flattenRoute53ResolverRuleTargetIps(targetAddresses []*route53resolver.Targ
 
 	return vTargetIps
 }
+
+func flattenOrganizationsOrganizationalUnits(ous []*organizations.OrganizationalUnit) []map[string]interface{} {
```

Comment:
Can you please move this function to the only place its called in `aws/data_source_aws_organizations_organizational_units.go`? We are trying to lower the usage of `aws/structure.go` for future refactoring work, thanks!